frontend/src/common/node.ts

```diff
@@ -84,15 +93,47 @@ export const mapCorrelationToName = (correlation: number) => {
 
 export const mapCorrelationToColor = (correlation: number) => {
   switch (correlation) {
-    case 2:
+    case 3:
       return '.600';
-    case 1:
+    case 2:
       return '.500';
-    case 0:
+    case 1:
       return '.400';
     default:
       return '.300';
   }
 };
+const isPosetiveConnection = (edge: D3Edge | GraphEdge, value: number): boolean => {
+  if (value === 2 && edge.correlation === 1) {
```

Comment:
I think this whole function can be simplified. Isn't it possible to just do `return edge.correlation > value`, with some rewriting?